### PR TITLE
Homepage | Logo - New UI

### PR DESCRIPTION
**Pull request by aayush-05**
**Screenshots**
1.
image
2.
image
3.
image
4.
image

### CODE REVIEW

**Comment by aayush-05** [comment]
@tachyons @nitin10s have a look at this.

File: app/assets/stylesheets/logixRelease.scss [hunk 1]
```diff
@@ -281,3 +281,93 @@ a.btn.btn-outline-danger {
 .subheading_teachers{
   text-align: left;
 }
+
+// Home page New UI
+.row_homepage {
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/assets/stylesheets/logixRelease.scss]: Done

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  text-decoration: none;
+}
+
+.btn-homepage2 {
+  border: 2px solid $teal;
+  border-radius: 5px;
+  margin: 10px;
+}
+
+.btn-homepage2:hover {
```
Comment (tachyons) [inline app/assets/stylesheets/home.scss]: use sass syntax for pseudo classes

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -73,3 +80,8 @@
   position: absolute;
   top: 35%;
 }
+
+.dropdown-item:hover {
```
Comment (tachyons) [inline app/assets/stylesheets/layout.scss]: use sass pseudo class syntax

File: app/assets/stylesheets/logixRelease.scss [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
-  [class*="display-4"] {
-    padding-top:10px;
-  }
-}
+// @media only screen and (max-width: 480px) {
+//   /* For mobile phones: */
+//   [class*="display-4"] {
+//     padding-top:10px;
+//   }
+// }
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.scss]: Remove if not needed instead of commenting it out

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -1,5 +1,5 @@
-<div class="container empty-div">
-</div>
+<!-- <div class="container empty-div">
+</div> -->
```
Comment (tachyons) [inline app/views/layouts/_footer.html.erb]: remove if not needed

File: app/assets/stylesheets/home.scss (hunk 2 above)
Comment (aayush-05) [reply app/assets/stylesheets/home.scss]: Done

File: app/assets/stylesheets/layout.scss (hunk 3 above)
Comment (aayush-05) [reply app/assets/stylesheets/layout.scss]: Done

File: app/assets/stylesheets/logixRelease.scss (hunk 4 above)
Comment (aayush-05) [reply app/assets/stylesheets/logixRelease.scss]: Done

File: app/views/layouts/_footer.html.erb (hunk 5 above)
Comment (aayush-05) [reply app/views/layouts/_footer.html.erb]: Added it till we make the footer sticky, then it can be removed.

**Comment by aayush-05** [comment]
These errors are conflicting @tachyons . Which one to follow ?
image

File: app/assets/stylesheets/home.scss [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+.a-homepage:hover {
+  color: $green;
+  text-decoration: none;
+}
+
+.btn-homepage2 {
+  border: 2px solid $teal;
+  border-radius: 5px;
+  margin: 10px;
+
```
Comment (tachyons) [inline app/assets/stylesheets/home.scss]:
```suggestion

```

File: app/assets/stylesheets/home.scss [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+.a-homepage:hover {
+  color: $green;
+  text-decoration: none;
+}
+
+.btn-homepage2 {
+  border: 2px solid $teal;
+  border-radius: 5px;
+  margin: 10px;
+  &:hover {
```
Comment (tachyons) [inline app/assets/stylesheets/home.scss]:
```suggestion

  &:hover {
```

File: app/assets/stylesheets/home.scss [hunk 8]
```diff
@@ -0,0 +1,92 @@
+// Home page New UI
+.row-homepage {
+  margin-bottom: 40px;
+  margin-top: 70px;
+
```
Comment (tachyons) [inline app/assets/stylesheets/home.scss]: ✂️

Comment (tachyons) [inline app/assets/stylesheets/home.scss]:
```suggestion
```

**Comment by aayush-05** [comment]
Thanks @tachyons . Please check it @nitin10s

**Comment by aayush-05** [comment]
Updated screenshots :
image
image
image

**Pull request merged**